Source/Data/BPMFMappings.txt

```diff
@@ -53372,6 +53387,8 @@
 懶漢 ㄌㄢˇ ㄏㄢˋ
 懶蟲 ㄌㄢˇ ㄔㄨㄥˊ
 懶骨頭 ㄌㄢˇ ㄍㄨˇ ㄊㄡˊ
+懶骨頭 ㄌㄢˇ ㄍㄨˇ ㄊㄡ˙
+懶骨頭 ㄌㄢˇ ㄍㄨˊ ㄊㄡˊ
```

Comment:
`/ㄍㄨˊ ㄊㄡˊ/` may be unnecessary because this combination probably doesn't fit the Beijing Mandarin tendency.

Reply:
I would keep this as it can be a common typing habit for some users.

Reply:
@xatier I'd argue that it's uncommon to type `/ㄍㄨˊ/` for 骨.

Reply:
I'm not sure if it's rare or not, but I've heard people pronounce this way in various occasions.

Reply:
@xatier I should probably clarify that more: I mean it's uncommon to type `/ㄍㄨˊ ㄊㄡˊ/` because natural pronunciation to me is `/ㄍㄨˊ ˙ㄊㄡ/` as the revised edition of MOE dictionary listed. So I should have put my 2nd comment like
> I'd argue that it's uncommon to type /ㄍㄨˊ/ for 骨 when next syllable is `/ㄊㄡˊ/`.

That being said, I also understand that this situation resembles making combination of tone sandhi (even though I tend to categorize that it's not tone sandhi and it's a specific pronounciation of bound morpheme, e.g., https://blog.udn.com/jangjnan/4026925).  So basically no strong opinions whether to have it or not.

I rest my case here.